README.md

```diff
@@ -1 +1,16 @@
-# hello-microverse
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+
+## Built With
+
+- HTML
+- CSS
+
```

Comment:
- [ ] [REQUIRED] I kindly suggest you add steps to run the project locally:
```
### EXAMPLE
## Getting Started
To get a local copy up and running, follow the steps in Setup below.

### Setup
If you installed git you can clone the code to your machine, or download a ZIP of all the files directly.

[Download the ZIP from this location](https://github.com/aeh1707/hello-microverse/archive/refs/heads/main.zip), or run the following [git](https://git-scm.com/downloads) command to clone the files to your machine:

```bash
git clone https://github.com/aeh1707/hello-microverse.git
```

`- Once the files are on your machine, open the _hello-microverse_ folder in [Visual Studio Code](https://code.visualstudio.com/).`
```

- With the files open in Visual Studio Code, press the **Go Live** button at the bottom of the window to launch the files with [Live Server](https://marketplace.visualstudio.com/items?itemName=ritwickdey.LiveServer).